Update project results (#207)
w3c/wai-about-wai: _about/projects/wai-guide/index.md

-   W3C Group Notes for [XR Accessibility User Requirements](https://www.w3.org/TR/xaur/), [RTC Accessibility User Requirements](https://www.w3.org/TR/raur/), and [Synchronization Accessibility User Requirements](https://www.w3.org/TR/saur/), to help address accessibility of immersive environments , real-time communication, and audio and visual media synchronization
-   On-going development of [ATAG Report Tool (ART)](https://www.w3.org/WAI/atag/report-tool/) to help promote accessibility of authoring tools
-   Launch of [Online Course: Introduction to Web Accessibility](https://www.w3.org/blog/2019/12/free-online-course-introduction-to-web-accessibility/) based on the curriculum developed through WAI-Guide

> -   Release of [WAI Curricula](https://www.w3.org/WAI/curricula/) with foundation, developer, and designer modules. [Content Author Modules](https://wai-curricula.netlify.app/curricula/content-author-modules/) in development)

Reviewer: ```suggestion
-   Release of [WAI Curricula](https://www.w3.org/WAI/curricula/) with foundation, developer, designer, and content author modules.
```

Author: This one relates to previous announcements. At the time of the announcement (which admittedly isn't included in the listing) the Content Author Modules were not released. On that basis, I probably wouldn't include this suggestion.

Having said all that we may end up just removing all the announcements!

## Project Results

### Supplemental resource framework redesign

> New styling was developed for use in Supplemental resources to:

Reviewer: ```suggestion
A new styling was developed for use in Supplemental resources to:
```

Author: I _think_ it would be 'A new style' and 'New styling'. Going to stick with the existing.

* improve the visual design for readability

> * improve navigation within a pages

Reviewer: ```suggestion
* improve navigation within pages
```

This was applied to:

* [Understanding for WCAG 2](/WAI/WCAG21/Understanding/)
* [Techniques for WCAG 2](/WAI/WCAG21/Techniques/)
* [ARIA Authoring Practices Guide](/WAI/ARIA/apg/)
* [Accessibility Conformance Testing Rules](/WAI/standards-guidelines/act/rules/)
* [Cognitive Accessibility Guidance](/WAI/WCAG2/supplemental/#cognitiveaccessibilityguidance)

### Expanded Technical Guidance

Technical guidance was expanded to better address current and emerging technologies. The specific areas covered were:

* [Design Guide for Cognitive Accessibility](/WAI/WCAG2/supplemental/#cognitiveaccessibilityguidance)
* [Immersive Environment requirements in WCAG3](/TR/xaur/)
* [Functional Needs in WCAG3](https://w3c.github.io/fast/)

### Video Training Materials

Created four sets of videos to support training of accessibility skills.

* [Web Accessibility Evaluation](/WAI/test-evaluate/) - Five animated videos that help explain the topic 'web accessibility evaluation'.
* [Stories of Web Users](https://www.youtube.com/watch?v=Z5dMdJzUy7w&list=PLhDEeYUfW02SJU-GQ1tagtChv7YmzSoQT) - Eight videos presenting stories of how people with disabilities use digital technology.

> * [Diverse Abilities and Barriers](https://www.youtube.com/watch?v=4UaUSbyVFjk&list=PLhDEeYUfW02SyKyI4cq-kpeGIBH6aTFHb) - Five videos that present the wide range of diversity of people and abilities, and highlights some of the types of barriers that people commonly encounter due to inaccessible design.

Reviewer: ```suggestion
* [Diverse Abilities and Barriers](https://www.youtube.com/watch?v=4UaUSbyVFjk&list=PLhDEeYUfW02SyKyI4cq-kpeGIBH6aTFHb) - Five videos that present the wide range of diversity of people and abilities, and highlight some of the types of barriers that people commonly encounter due to inaccessible design.
```

* [Tools and Techniques](https://www.youtube.com/watch?v=4xF23iGIAPE&list=PLhDEeYUfW02QfCwXHtNKyqi31UPuuaAlO) - Four videos that present techniques and tools that people with disabilities use to interact with the Web.

### Accessibility Curricula

> Created a framework to support the development of courses on digital accessibility. The curricula covered four core modules:

Reviewer: ```suggestion
Created a framework to support the development of courses on digital accessibility, as well as the inclusion of digital accessibility in courses for related areas. The curricula covered four core modules:
```

> * [Foundation Modules](/WAI/curricula/foundation-modules/) - designed to provide a general introduction to web accessibility to any students with a basic understanding of computers and the Web.

Reviewer: ```suggestion
* [Foundation Modules](/WAI/curricula/foundation-modules/) - helps introducing web accessibility to any students with a basic understanding of computers and the Web.
```

> * [Developer Modules](/WAI/curricula/developer-modules/) - provides guidance on creating courses that explain accessible coding for developers.

Reviewer: ```suggestion
* [Developer Modules](/WAI/curricula/developer-modules/) - helps create courses that explain accessible coding for developers.
```

> * [Designer Modules](/WAI/curricula/designer-modules/) - provides guidance on creating courses that explain how design principles and practices relate to accessibility.

Reviewer: ```suggestion
* [Designer Modules](/WAI/curricula/designer-modules/) - helps create courses that explain how design principles and practices relate to accessibility.
```

> * [Content Author Modules](/WAI/curricula/content-author-modules/) - provides guidance on creating courses that explains how to create clear content and accessible authoring techniques.

Reviewer: ```suggestion
* [Content Author Modules](/WAI/curricula/content-author-modules/) - helps create courses that explain how to create clear content and discusses accessible authoring techniques.
```

* [For No-Code Tools](/WAI/standards-guidelines/atag/no-code/) - provides an introduction to help make no-code website tools accessible to creators, designers, and end users with disabilities.
* [For Social Media Platforms](/WAI/standards-guidelines/atag/social-media/) - provides an introduction to help make social media platforms accessible to creators and users with disabilities

### Authoring Tools List

[Authoring Tools List](/WAI/tools-list/authoring/) - Provides a resource that lists authoring tool vendor submitted information about the accessibility of their tools.

### Authoring Tools Conformance Evaluation Report Tool

[ATAG Report Tool](/WAI/atag/report-tool/) leads the user step-by-step through the requirements defined by the W3C Authoring Tool Accessibility Guidelines and allows the user to record the results for each requirement. The tool generates a report based on this input by the user in HTML and in JSON formats.

### New User Needs

Identified and explored accessibility gaps in new standards for emerging technologies. This work developed four new user requirements documents:

> * [XR Accessibility User Requirements](/TR/xaur/) - covers user needs and requirements for people with disabilities when using virtual reality or immersive environments, augmented or mixed reality and other related technologies (XR).

Reviewer: ```suggestion
* [XR Accessibility User Requirements](/TR/xaur/) - covers user needs and requirements for people with disabilities when using virtual reality or immersive environments, augmented or mixed reality, and other related technologies (XR).
```
